### PR TITLE
Move TestApplication.Wcf.Shared DTOs to TestApplication.Wcf.Client.Core

### PR DESCRIPTION
**Pull request by pellared**
## Why

Leftover after https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1821

## What

Move TestApplication.Wcf.Shared DTOs to TestApplication.Wcf.Client.Core

## Tests

CI

### CODE REVIEW

**Review by dszmigielski — APPROVED** [review]
Please check if documentation is up to date after this change

**Comment by pellared** [comment]
> Please check if documentation is up to date after this change

Checked

File: test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@
 using System.ServiceModel;
 using System.Threading.Tasks;
 
-namespace TestApplication.Wcf.Shared;
+namespace TestApplication.Wcf.Client.Core;
```
Comment (Kielek) [inline test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs:20]: In contrib repo there were a change Client.Core -> Client.DotNet.

Any chance to include into this PR?

Ref.https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/857

**Review by Kielek — APPROVED** [review]

File: test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs (hunk 1 above)
Comment (pellared) [reply test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs:20]: I will create an issue and I will do it in next PR once I have some time.

Comment (pellared) [reply test/test-applications/integrations/TestApplication.Wcf.Client.Core/IStatusServiceContract.cs:20]: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1874

**Pull request merged**